mantidproject/mslice
# 129 option change resolution

Description of work.
Added functionality for users to choose dpi when saving an image
**To test:**


Open mslice, perform a cut or slice. Click the save image icon. In the save dialog, opt to save as an image by choosing a .pdf or .png extension. Click save. Another dialog should come up asking what dpi the user wants. 30 is the minimum and 300 is the default.


Fixes #129 .
